tests/setup_machine.sh

```diff
@@ -91,7 +91,6 @@ common_packages=(
 	socat
 	sudo
 	tidy
-	time
```

Comment:
Why was it necessary to delete this common package? Just a little doubt.

Reply:
It's a mistake, thanks for noticing :)